pcsx2/CMakeLists.txt

```diff
@@ -646,7 +646,7 @@ include_directories(
 ### Generate the resources files
 file(MAKE_DIRECTORY ${res_bin})
 
-add_custom_command(OUTPUT  "${res_bin}/Dualshock.h"  COMMAND perl ${CMAKE_SOURCE_DIR}/linux_various/hex2h.pl "${res_src}/Dualshock.jpg" "${res_bin}/Dualshock" )
+add_custom_command(OUTPUT  "${res_bin}/Logo.h"  COMMAND perl ${CMAKE_SOURCE_DIR}/linux_various/hex2h.pl "${res_src}/Logo.png" "${res_bin}/Logo" )
```

Comment:
Remove this line and add Logo to the loop below. This line exists because of the extension .jpg (versus .png for others images)